pkg/cmdrunner/shellrunner.go

```diff
@@ -92,15 +93,15 @@ func (sr *ShellRunner) Run(runOptions *RunOptions, format string, vars ...interf
 			"exitCode", runResult.ExitCode,
 			"err", err)
 
-		err = errors.Wrapf(err, "stdout:\n%s\nstderr:\n%s", runResult.Output, runResult.Stderr)
-
-		return runResult, err
+		return runResult, errors.Wrapf(err, "stdout:\n%s\nstderr:\n%s", runResult.Output, runResult.Stderr)
 	}
 
-	sr.logger.DebugWith("Command executed successfully",
-		"output", runResult.Output,
-		"stderr", runResult.Stderr,
-		"exitCode", runResult.ExitCode)
+	if !runOptions.SkipLoggingBeforeAfterExecute {
```

Comment:
why is `SkipLoggingBeforeAfterExecute` a thing?

Reply:
to avoid spamming the stdout for periodically commands (such as docker version)
changed to `LogOnlyOnFailure`